neural_lam/models/ar_model.py

```diff
@@ -422,18 +464,34 @@ def plot_examples(self, batch, n_examples, prediction=None):
             prediction, target, _, _ = self.common_step(batch)
 
         target = batch[1]
+        time = batch[3]
```

Comment:
In the docstring would it make sense to describe what is expected in each index of the list?

Reply:
True, although one should probably infer that from `WeatherDataset.__getitem__`. I would prefer to not change that here, again since this code will likely change.